src/graphics/program-lib/programs/standard.js

```diff
@@ -305,13 +305,18 @@ const standard = {
             code.append(this._addMap("diffuse", "diffusePS", options, litShader.chunks));
             func.append("getAlbedo();");
 
+            if (options.refraction) {
+                decl.append("float dIor;");
+                code.append(this._addMap("refractionIndex", "iorPS", options, litShader.chunks));
```

Comment:
This function assumes a full map - so you need to add all the iorPS members to material - iorPSMap, iorPSUv, iorPSTiling, iorPSOffset etc and then handle them in the shader.

Reply:
I think you mean `refractionIndexMap`, `refractionMapUv`, etc, right?

Reply:
Yes!

Though I see from https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_ior/README.md that refraction index is a per-material constant (not texture or vertex driven).

@GSterbrant is it even necessary to have a customisation chunk for this? Seems like ior is literally just a float per material. User can set that on javascript - no need for customising chunks etc.